pagopa/io-app
# chore(IT Wallet): [SIW-950] Add ID Pay initiatives cards in new wallet section

> [!WARNING]
> This PR depends on https://github.com/pagopa/io-app/pull/5619

## Short description
Include a summary of the changes.

## List of changes proposed in this pull request
- Added SVG asset for the ID Pay wallet card
- Added ID Pay cards in the wallet section

## How to test
You should use the [io-dev-api-server ](https://github.com/pagopa/io-dev-api-server).
Within the Profile section, enable the **ID Pay** and **New wallet section** feature flags. You should be able to see and interact with ID Pay cards in the wallet section

## Preview
<img src="https://github.com/pagopa/io-app/assets/6160324/7053bced-ad82-4b96-9dd6-3dbef4a7ab09" width="200" />
